src/plugins/customStyles/index.js

```diff
@@ -0,0 +1,32 @@
+import { CUSTOM_BLOCK_STYLES,CUSTOM_TEXT_STYLES } from './constants';
+import Immutable from 'immutable';
+import styles from './customStyles.css';
+
+const createCustomStylesPlugin = () => {
+
+    const blockRenderMap = Immutable.Map(CUSTOM_BLOCK_STYLES);
+    const customStyleMap = CUSTOM_TEXT_STYLES;
+
+    const blockStyleFn = (contentBlock) => {
+        let style = [];
+
+        const type = contentBlock.getType();
+        for(var key in CUSTOM_BLOCK_STYLES) {
+            if(CUSTOM_BLOCK_STYLES.hasOwnProperty(key)) {
```

Comment:
pour tout te dire j'ai vu ce type de check dans un code et je me suis demandé si dans certains cas le for ne va pas me retourner une fonction ou quelque chose qui n'est pas une "property".

je ne maitrise pas ce que fait exactement hasownproperty ;)